src/main/kotlin/no/elhub/devxp/kotlin-service.gradle.kts

```diff
@@ -6,5 +6,31 @@ package no.elhub.devxp
 plugins {
     id("no.elhub.devxp.kotlin-core")
     id("com.jfrog.artifactory")
+    id("maven-publish")
     id("application")
 }
+
+/*
+ * Publishing
+ */
+artifactory {
+    clientConfig.isIncludeEnvVars = true
+
+    publish {
+        contextUrl = project.findProperty("artifactoryUri")?.toString() ?: "https://jfrog.elhub.cloud/artifactory"
+        repository {
+            repoKey = project.findProperty("artifactoryRepository")?.toString() ?: "elhub-mvn-dev-local"
```

Comment:
Should this be elhub-mvn-release-local and not dev? (I thought dev-repositories existed for testing stuff out, but was not intended to be used for "real" stuff)

EDIT: Right, this is just a fallback-value i guess!